src/test-support/mocks/src/dataplanes+insights.ts

```diff
@@ -1,10 +1,9 @@
 import type { EndpointDependencies, MockResponder } from '@/test-support'
 export default ({ fake, pager, env }: EndpointDependencies): MockResponder => (req) => {
-  const params = req.params
   const { offset, total, next, pageTotal } = pager(
     env('KUMA_DATAPLANE_COUNT', `${fake.datatype.number({ min: 1, max: 1000 })}`),
     req,
-    `/meshes/${params.mesh}/dataplanes+insights`,
+    '/dataplanes+insights',
```

Comment:
I spotted this was wrong so fixed this up whilst I was here, would be nice for `Pager` to automatically figure these out in the future.

Reply:
I think there's a couple. One is at root based and one is mesh based:

https://github.com/kumahq/kuma-gui/blob/1c7311e7d093397b853ce7f50dd224f7daf13493/src/services/kuma-api/KumaApi.ts#L134-L140

This one is the root one. Not totally sure if we still use the root based one in the GUI. 👀 

At first glance it looks like we don't so we could completely remove the root one (and any other similar root ones).

Reply:
Ok feel free to open an issue to figure this out later and ignore this comment :)